react/src/lib/components/SubsurfaceViewer/layers/map/mapLayer.stories.tsx

```diff
@@ -1263,6 +1263,7 @@ const ContourLinesStory = (props: {
     contourOffset: number;
     zContourInterval: number;
     propertyContourInterval: number;
+    marginPixels: number;
```

Comment:
Suggest creating a new story to demonstrate the margins. If there are regressions specific to margins in the future it will be isolated to that story.

Would also be good to look into improving cypress test coverage. We have some tests already that you could look into. Or start using this storybook addon: https://storybook.js.org/docs/react/writing-tests/visual-testing

Reply:
Added a story at the top level ("Matrix")
Also the cyprus test hopefully can go in a separate PR as I need some time to figure out how cypress tests are made and run.